test/basic-querying.test.js

```diff
@@ -958,6 +958,25 @@ describe('basic-querying', function() {
         null, // databases representing `undefined` as `null` (e.g. SQL)
       ]);
     });
+
+    describe('check __parent relationship in embedded models', () => {
+      it('should fill the parent in embedded model', async () => {
+        const [user] = await User.find({where: {name: 'John Lennon'}});
```

Comment:
Nitpick:

```suggestion
        const user = await User.findOne({where: {name: 'John Lennon'}});
```

Same comment applies to other new tests below too.

Reply:
Ok I will change the tests reference

Reply:
Done